test/src/main/java/io/strimzi/test/TestUtils.java

```diff
@@ -125,6 +125,7 @@ public static long waitFor(String description, long pollIntervalMs, long timeout
             try {
                 result = ready.getAsBoolean();
             } catch (Exception e) {
+                LOGGER.info("Exception waiting for {}, {}", description, e.getMessage());
```

Comment:
I am not sure if this is a good solution. We had discussion around it and this exception should be printed only when the wait fails. This info log can easilly log a lot of same lines without any useful info from it. I think we maybe should try to execute same code again on line 136 (before `onTimeout.run()`) and print the exception here.

Reply:
I see your point about logging a lot of lines and I agree. The situation I encountered was that in my initial setup `kubectl` was not found and therefore the process could never be started. The exception message clearly indicated what was wrong, but based on the build output on the console, the process seemed to be stuck. Ideally for situations like that, a user would be notified for fatal errors and be able to cancel the build, etc.

What are your thoughts on a solution where the message is logged only the first time it is encountered?

Reply:
Ye sounds like a good option. The first and last exception on the info log level would be great.